### PR TITLE
Update direct-routing-enable-users.md

### PR DESCRIPTION
**Pull request by gosepulv**
There have been examples of admins confusing the required configuration with OnPremLineUriManuallySet: False where the article sets an expectaiton of True. The attribute is not modifiable manually and only setting OnPremLineUri tel:+xxxxxxxxxxx will set OnPremLineUriManuallySet: True.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-enable-users.md [hunk 1]
```diff
@@ -99,7 +99,9 @@ After you have created the user and assigned a license, the next step is to conf
     ```PowerShell
     Set-CsUser -Identity "<User name>" -EnterpriseVoiceEnabled $true -HostedVoiceMail $true -OnPremLineURI tel:<phone number>
     ```
-    
+    > [!NOTE]
+    > By executing this cmdlet successfully to configure the user's phone number for Direct Routing, will be set OnPremLineUriManuallySet to True.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-enable-users.md:103]:
```suggestion
    > By executing this cmdlet successfully to configure the user's phone number for Direct Routing, OnPremLineUriManuallySet will be set to True.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @gosepulv, kindly commit the suggested change. Many thanks.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
this PR has conflicts.  unable to merge until conflicts are resolved.

**Comment by scanum** [comment]
@gosepulv Could you please review conflicts? Thanks

**Comment by CarolynRowe** [comment]
This PR is over 1 year old.  I'm concerned about merging something this old into the updated doc set.  If this is still an issue, the submitter should file another PR.  I am closing this one.

**Pull request closed without merging**